samples/samples-js/AddProductExtraColumns/index.js

```diff
@@ -0,0 +1,18 @@
+module.exports = async function (context) {
```

Comment:
This doesn't really belong in the samples folder since it's showing incorrect behavior. For now let's just only add JS versions of the functions in the samples folder - any in the test folder just skip and we'll add those separately when we add the tests later.